packages/schema/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "author": "Google Labs Team",
+  "devDependencies": {
+    "@google-labs/tsconfig": "^0.0.1",
+    "@types/node": "^20.11.10",
+    "ajv": "^8.12.0",
+    "ajv-formats": "^2.1.1",
+    "wireit": "^0.14.4"
+  },
+  "files": [
+    "breadboard.schema.json"
+  ],
+  "license": "Apache-2.0",
+  "name": "@google-labs/breadboard-schema",
+  "scripts": {
+    "prepack": "wireit",
+    "validate": "wireit"
+  },
+  "version": "1.0.0",
+  "wireit": {
+    "prepack": {
```

Comment:
Smart!

Reply:
which bit?

Reply:
Oh just using prepack to enforce that it validates so that it's impossible to publish  with validation errors.

Reply:
I'm always a fun of hooking into lifecycle stages. especially if it means not having to install git hooks